pages/addCommunity.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from 'react';
+
+import Button from '../components/button/Button';
+import Card from '../components/card/Card';
+import request from '../modules/request';
+import { Form, Input, Label } from '../styles/AddCommunity.styles';
+import { AppWrapper, GlobalStyle, Navigation } from '../styles/index.styles';
+
+const AddCommunity = (): JSX.Element => {
+  const [title, setTitle] = useState('');
+  const [url, setUrl] = useState('');
+  const [members, setMembers] = useState('');
+  const [image, setImage] = useState('');
+  const isEnabled =
```

Comment:
The way you wrote that, isEnabled is evaluated only once. If a value changes like that, we should use a useEffect. Let's do it together tomorrow.

Reply:
Done!